package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ginger.io/jay-z",
-  "version": "0.0.13",
+  "version": "0.0.14",
```

Comment:
Given this version is already published, you'll want to open another PR later to bump the version (you can't overwrite npm packages, and generally shouldn't anyways).